altair/utils/tests/test_utils.py

```diff
@@ -106,3 +106,61 @@ def test_sanitize_dataframe_infs():
     df_clean = sanitize_dataframe(df)
     assert list(df_clean.dtypes) == [object]
     assert list(df_clean['x']) == [0, 1, 2, None, None, None]
+
+
+def test_sanitize_nullable_integers():
```

Comment:
Rather than silently passing if there is a type error, it would be better to explicitly skip this test for older versions of pandas. The cleanest way would be to add a decorator like:
```python
@pytest.mark.skipif(not hasattr(pd, "Int64Dtype"), "Nullable integers not supported in pandas v{}".format(pd.__version__))
def test_sanitize_nullable_integers():
    ...
```
and then remove the ``try...except`` statement.

Similar for the string test below.